lib/components/link.vue

```diff
@@ -52,16 +52,12 @@
             },
             _to() {
                 if (!this.routerAvailable || this.disabled) {
```

Comment:
Then this condition won't happen  as `routerAvailable` is false when no `to` prop exists :))